README.md

```diff
@@ -1,786 +1,104 @@
 # Citation File Format (CFF)
+
 [![Build Status](https://github.com/citation-file-format/citation-file-format/workflows/testing/badge.svg)](https://github.com/citation-file-format/citation-file-format/actions/workflows/testing.yml)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1003149.svg)](https://doi.org/10.5281/zenodo.1003149)
 [![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
 
 The Citation File Format (CFF) is a human- and machine-readable file format in YAML 1.2 which provides
```

Comment:
contracting the human and machine readability is a bit of an antipattern for English. Maybe we can end the sentence with something like _readable by humans as well as machines_, like so? 

```suggestion
The Citation File Format specifies how users can provide citation metadata that is easy to read by humans as well as by machines.
```

Reply:
```suggestion
The Citation File Format specifies how users can provide citation metadata for software and datasets that is easy to read by humans as well as by machines.
```